justfile

```diff
@@ -72,6 +72,23 @@ benchmark-pallet chain="polimec-paseo-local"  pallet="pallet-dispenser":
       --output=pallets/{{ replace(pallet, "pallet-", "") }}/src/weights.rs \
       --template=./.maintain/frame-weight-template.hbs
 
+
+benchmark-extrinsics pallet="pallet-funding" extrinsics="*" :
+    cargo run --features runtime-benchmarks --profile=production -p polimec-node benchmark pallet \
```

Comment:
I would point to the actual binary in `target/production`, without using `cargo run`. Otherwise, consequential runs could be quite slow

Reply:
I think you mean subsequential :)
I think running cargo run several times without changing the code does not produce a new compilation every time, but I can change it if you like